src/deepsparse/yolo/pipelines.py

```diff
@@ -192,7 +192,9 @@ def process_inputs(self, inputs: YOLOInput) -> List[numpy.ndarray]:
                 image = cv2.imread(image)
 
             image = self._make_channels_last(image)
-            image = cv2.resize(image, dsize=tuple(reversed(self.image_size)))
+            if image.ndim < 4:
+                # Assume a batch is of the correct size already
```

Comment:
Wait why is this a safe assumption?

Reply:
images need to be reshaped to get into a batch, assumption here is that preprocessing has already been run to get it into a batch. need to add appropriate documentation @rahul-tuli

Reply:
Can you suggest the wording?

Reply:
This assumption was already documented on lines 174, 175; changed the wording a bit. Let me know if it looks good

Reply:
was thinking the documentation should be at the Schema docstring level